### PR TITLE
Add optional "Help" button to bottom-left of wizard window

### PR DESCRIPTION
**Pull request by herzbube**
The "Help" button is not visible by default. If it is made visible by
a wizard implementation (property IsHelpVisible), it displays with the
label "Help" and is enabled by default. If necessary the implementation
can disable the button (property CanShowHelp). When the user clicks the
button the event nothing happens by default. It is the responsibility
of the implementation to handle the event, either by reacting to the
event HelpShown or by overriding ShowHelpAsync().

"Creative" wizard implementations can make the button display a
different label than "Help" and do something completely else. The only
thing that is fixed is the button's location in the bottom-left of
the wizard window.

---

@GeertvanHorrik I promised/announced this PR a few months ago on Gitter, so here it is, finally. Hope you still find it useful. Just so you know: I made the localization of the "Help" label by visiting google.com in the various languages, doing a search and then looking at the translation of "Help" in the bottom left of the browser window.

It will be the last PR for a while, I think, as we are now happy with Orc.Wizard's feature set. It would be extremely cool if you found the time to make an official release so that we could integrate an official package instead of a custom build.

Thanks
Patrick

### CODE REVIEW

**Comment by GeertvanHorrik** [comment]
Thanks a lot for the PR, they are much appreciated. As soon as Catel becomes stable, we will release this component as stable as well. We are currently ramping up Catel and all the other Orc.* controls, then we will deploy all the components as stable.

File: src/Orc.Wizard/Orc.Wizard.Shared/ViewModels/WizardViewModel.cs [hunk 1]
```diff
@@ -130,6 +132,18 @@ private bool OnCancelCanExecuteAsync()
         {
             return Wizard.CanCancel;
         }
+
+        public TaskCommand ShowHelp { get; set; }
+
+        private async Task OnShowHelpExecuteAsync()
+        {
+            await Wizard.ShowHelpAsync();
```
Comment (GeertvanHorrik) [inline src/Orc.Wizard/Orc.Wizard.Shared/ViewModels/WizardViewModel.cs]: You can use this instead (saves another state machine at compilation time and task object at runtime):

```
private Task OnShowHelpExecuteAsync() 
{
    return Wizard.ShowHelpAsync();
}
```

**Comment by herzbube** [comment]
OK, thanks for the heads-up.

**Comment by GeertvanHorrik** [comment]
Build - 1.1.0-unstable.35 has been released

**Pull request merged**